docs/user/exams/students_guide.rst

```diff
@@ -25,11 +25,11 @@ Prerequisites
 
     - **Java IDE with JDK 17**
```

Comment:
please change line 24 above to:

- The following prerequisites are only required if your exam contains Java programming exercises: